packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -15,13 +17,11 @@ export default function({ types: t }) {
 
   return {
     pre(file) {
-      const moduleName = getRuntimeModuleName(this.opts);
+      const moduleName = getRuntimeModuleName(options);
```

Comment:
This function has `options` in scope, so it doesn't need the argument at all. Can also move the `opts.moduleName` reference into your destructuring.